### ISSUE TITLE
Problem with complete reference/citation functionality, OSX 10.11.1, ST3 B3083

### ISSUE THREAD

**Issue by mrscholz** [SublimeText/LaTeXTools#615]
It seems that the described functionality to open a selection whenever typing \ref{  or \cite{ in a LaTeX document is not working. Whenever I type, e.g., \ref LaTeXTools won't let me insert the opening curly brace and nothing happens. I first thought its related to the german keyboard that I'm using but changing to a US layout did not help.

**Comment by r-stein** [comment]
Can you open the console `View > Show Console` and check the output when pressing the `{` key.

**Comment by mrscholz** [comment]
Yes, but it does not show any message at all. Do I have to turn on a more *verbose* mode?

**Comment by r-stein** [comment]
Type into the console
``` python
sublime.log_commands(True)
```
to show which commands are executed.
This should be
``` javascript
command: latex_ref_cite {"insert_char": "{"}
```

**Comment by mrscholz** [comment]
confirmed: 
```command: latex_ref_cite {"insert_char": "{"}```

**Comment by r-stein** [comment]
The output should be something like
```
command: latex_ref_cite {"insert_char": "{"}
a number
True True
Dispatching ref
a number
TEX root: 'your/tex/root.tex'
Searching file: 'your/tex/root.tex'
```

**Comment by mrscholz** [comment]
There is no further output beyond  ```command: latex_ref_cite {"insert_char": "{"}```

**Comment by r-stein** [comment]
Then Sublime Text does not know the command and restarting ST could be sufficient.

Otherwise you can open the command definition file by pasting this into the console:
``` py
import os; window.open_file(os.path.join(sublime.packages_path(), "LaTeXTools", "latex_ref_cite_completions.py"))
```
Save it and check if errors occur in the console output.

**Comment by mrscholz** [comment]
While a restart did not help (this I tried before), **saving the command definition file somehow fixed it!** 
Thanks a lot!

**Comment by r-stein** [comment]
Saving the file also reloads the command, but I don't understand why your ST did not load it automatically.

**Comment by mrscholz** [comment]
Me neither (of course). Well, I experimented with other Latex packages like LaTeXing. In between my experiments, one of your recent updates appeared and that might indeed correlate with the time when the functionality stopped working. I first figured out that something was wrong when I was unable to put a ```{``` behind a ```\ref```. I started disabling LaTeX related packages and found the LaTeXTools to be responsible. I restarted ST several times, however, I am not 100% sure whether the LTTools were enabled at one of those restarts. However, since I know that some of the functionality that comes with packages requires a restart to work, this is usually the first thing I try if something is not working as expected.

Nevertheless, this may count as another confirmation that 99% of the issues related to computers sit in front of it and I apologize for the inconvenience.
